torchvision/prototype/transforms/functional/_dispatch.py

```diff
@@ -0,0 +1,448 @@
+    "resized_crop",
+    "erase",
+    "mixup",
+    "cutmix",
+    "affine",
+    "rotate",
+    "adjust_brightness",
+    "adjust_saturation",
+    "adjust_contrast",
+    "adjust_sharpness",
+    "posterize",
+    "solarize",
+    "autocontrast",
+    "equalize",
+    "invert",
+]
+
+
+@Dispatcher
+def horizontal_flip(input: T) -> T:
+    """ADDME"""
+    pass
+
+
+@horizontal_flip.implements(features.Image)
+def _horizontal_flip_image(input: features.Image) -> features.Image:
+    output = F.horizontal_flip_image(input)
+
+    return features.Image.new_like(input, output)
+
+
+@horizontal_flip.implements(features.BoundingBox)
+def _horizontal_flip_bounding_box(input: features.BoundingBox) -> features.BoundingBox:
+    intermediate_format = BoundingBoxFormat.XYXY
+    converted_input = F.convert_bounding_box_format(input, old_format=input.format, new_format=intermediate_format)
+
+    output = F.horizontal_flip_bounding_box(converted_input, image_size=input.image_size)
+    output = F.convert_bounding_box_format(output, old_format=intermediate_format, new_format=input.format)
+
+    return features.BoundingBox.new_like(input, output)
+
+
+@Dispatcher
+def resize(
+    input: T,
+    *,
+    size: List[int],
+    interpolation: InterpolationMode = FEATURE_SPECIFIC_DEFAULT,  # type: ignore[assignment]
+    max_size: Optional[int] = None,
+    antialias: Optional[bool] = None,
+) -> T:
+    """ADDME"""
+    pass
+
+
+@resize.implements(features.Image, pil_kernel=_F.resize)
+def _resize_image(
+    input: features.Image,
+    *,
+    size: List[int],
+    interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+    max_size: Optional[int] = None,
+    antialias: Optional[bool] = None,
+) -> features.Image:
+    output = F.resize_image(input, size=size, interpolation=interpolation, max_size=max_size, antialias=antialias)
+
+    return features.Image.new_like(input, output)
+
+
+@resize.implements(features.BoundingBox)
+def _resize_bounding_box(input: features.BoundingBox, *, size: List[int], **_: Any) -> features.BoundingBox:
+    intermediate_format = BoundingBoxFormat.XYXY
+    converted_input = F.convert_bounding_box_format(input, old_format=input.format, new_format=intermediate_format)
+
+    output = F.resize_bounding_box(converted_input, old_image_size=input.image_size, new_image_size=size)
+    output = F.convert_bounding_box_format(output, old_format=intermediate_format, new_format=input.format)
+
+    return features.BoundingBox.new_like(input, output, image_size="size")
+
+
+@resize.implements(features.SegmentationMask)
+def _resize_segmentation_mask(
+    input: features.SegmentationMask,
+    *,
+    size: List[int],
+    interpolation: InterpolationMode = InterpolationMode.NEAREST,
+    max_size: Optional[int] = None,
+    antialias: Optional[bool] = None,
+) -> features.SegmentationMask:
+    output = F.resize_segmentation_mask(
+        input, size=size, interpolation=interpolation, max_size=max_size, antialias=antialias
```

Comment:
We typically use nearest interp mode for segmenation masks resize and there is no antialias option for nearest mode. I think we can remove `antialias` argument.

Reply:
In that case, we should remove it from the kernel:

https://github.com/pytorch/vision/blob/7cffef67e251ab96663c3970056ad471db2aa6ed/torchvision/prototype/transforms/functional/_geometry.py#L53-L59

But that would mean one could set the interpolation mode to something else manually and would have no option to set antialias. Should we also hardcode the interpolation mode?

Reply:
Yes, I'd say there is less sense to use linear interpolation mode for masks as it would produce artificial target classes on resizing.